### PR TITLE
adds group appointment request

### PR DESCRIPTION
**Pull request by paulypeter**
I added a query parameter for fetching group appointments.

### CODE REVIEW

File: src/api_wrapper.py [hunk 1]
```diff
@@ -13,10 +13,19 @@
...
+def fetch_api(
+        plz: int,
+        birthdate_timestamp: int = None,
+        group_size: int = None,
+        max_retries: int = 10,
+        sleep_after_error: int = 30,
+        sleep_after_shadowban: int = 300
+    ) -> any:
+    url = f"https://www.impfportal-niedersachsen.de/portal/rest/appointments/findVaccinationCenterListFree/{plz}?stiko="
     if birthdate_timestamp is not None:
```
Comment (sibalzer) [inline src/api_wrapper.py:25]: could be shorten to if birthdate_timestamp:

File: src/api_wrapper.py [hunk 2]
```diff
@@ -13,10 +13,19 @@
...
-        url += f"?stiko=&count=1&birthdate={int(birthdate_timestamp)*1000}"
+        url += f"&count=1&birthdate={int(birthdate_timestamp)*1000}"
+    elif group_size is not None:
```
Comment (sibalzer) [inline src/api_wrapper.py:27]: same here e.g. elif group_size:

**Review by sibalzer — CHANGES_REQUESTED** [review]

File: src/api_wrapper.py (hunk 1 above)
Comment (paulypeter) [reply src/api_wrapper.py]: Done (ff13509)

File: src/api_wrapper.py (hunk 2 above)
Comment (paulypeter) [reply src/api_wrapper.py]: Done (ff13509)

**Review by sibalzer — APPROVED** [review]

**Comment by paulypeter** [comment]
This PR closes #39

**Pull request merged**